docs/pages/access-controls/access-graph/gitlab.mdx

```diff
@@ -0,0 +1,119 @@
+Access Graph, ensuring that it remains updated with the latest information from your GitLab instance.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources imported and their relationships, crafting a
+graphical representation thereof.
+
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v14.3.20/v15.3.1/v16.0.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.21.4 or later.
+Check [Access Graph page](../teleport-policy/getting-started-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable
+from Teleport Auth Service.
+
+## Step 1/3. Create GitLab token
+
+To set up the GitLab integration, you'll need to create a GitLab token with the following permissions:
+
+- `read_api`
+
+Navigate to your GitLab instance, access the User Settings, and select the Access Tokens option.
+Create a new token with the `read_api` scope and copy the generated token.
+
+The importer will use this token to fetch the necessary resources from your GitLab instance.
+
+<Notice type="warning">
+
+    The GitLab importer will only fetch resources that the token has access to. Ensure that the user associated with
+    the token has the necessary permissions to access/view the resources you want to import.
+
+    If you're using a GitLab.com account, the importer will only fetch the users who are part of the organization or
+    have access to the projects that the token can access.
+
+    If you're using a self-hosted GitLab instance, the importer will fetch all resources that the token has access to,
+    including all users who are part of the instance.
+
+</Notice>
+
+The token will be used in the next step to configure the GitLab Sync integration.
+
+## Step 2/3. Set up Access Graph GitLab Sync
+
+To initiate the setup wizard for configuring GitLab Sync, access the Teleport UI,
+navigate to the Management tab, and choose the Access Graph option within the
+Permission Management section.
+
+In the Access Graph page, you'll notice a button labeled `Integrations`. Click on it to
+to access the Integrations page. On the Integrations page, click on the `Setup` button next to the GitLab integration.
+
+You'll be prompted to provide the GitLab token created in Step 1 and the GitLab instance domain.
+Once the token is successfully validated, you'll be able to see the resources imported in Access Graph.
+
+## Step 3/3. View GitLab resources in Access Graph
+
+After the GitLab resources are imported, you can view them in the Access Graph page.
+The graph representation will show the relationships between users, groups, and projects within your GitLab instance.
+
+Users can have permissions to access a Group or Project. When a user has access to a Group, they inherit permissions
+to all projects and subprojects within that group.
+
```

Comment:
Can we give a few examples of queries users can run to analyze Gitlab access paths?